packages/@react-aria/listbox/src/useListBoxSection.ts

```diff
@@ -47,10 +47,10 @@ export function useListBoxSection(props: AriaListBoxSectionProps): ListBoxSectio
     },
     headingProps: heading ? {
       // Techincally, listbox cannot contain headings according to ARIA.
-      // We hide the heading from assistive technology, and only use it
-      // as a label for the nested group.
+      // We hide the heading from assistive technology, using role="presentation",
+      // and only use it as a label for the nested group.
```

Comment:
Is this more correct?
```suggestion
      // and only use it as a visual label for the nested group.
```